src/ansys/simai/core/data/workspaces.py

```diff
@@ -20,17 +20,21 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import warnings
 from pprint import pformat
-from typing import Any, BinaryIO, Dict, List, Optional, Union
+from typing import TYPE_CHECKING, Dict, List, Optional, Union
 
 from ansys.simai.core.data.base import DataModel, Directory
 from ansys.simai.core.data.types import File, Identifiable, get_id_from_identifiable
 
+if TYPE_CHECKING:
+    from typing import Any, BinaryIO
```

Comment:
I don't understand why the typing changes in that file and why are those in `if TYPE_CHECKING` ?
Not a fan of typing in quotes, will make it more annoying to refactor once we can switch to `dict`/`list`...

Reply:
Yeah I just forgot those after testing stuff